bk-api/test_bk_api.py

```diff
@@ -139,6 +139,30 @@ def test_log_insert_fail(client):
     assert 'error'  in result
 
 
+def test_vsn_insert(client):
```

Comment:
I think we should have a test for `register_add_vsn_event(node_id, vsn)` to make sure you can set a a few VSNs and read them back out?

This is more like integration testing and I'm not following exactly when and how these tests are run.

Maybe we can start working towards better defining the interfaces around the:
* Database
* API
* Processes like "deploy WES" and "fetch VSN"

With clean boundaries, it will be easy for us to just write a unit test like this for just the database portion.